coding.md

```diff
@@ -132,7 +155,7 @@ No space should be used after the following unary operators:
 The `sizeof` and `typeof`are treated as functions and are to be used in accordance to the following notation:
 
 >
-    sizeof(x)  
+    sizeof(x)
     typeof(x)
```

Comment:
note: typeof is GCC (and every modern compiler) extension, it's in standard from C23. 
However it's needed for many constructs.